src/core/Entry.cpp

```diff
@@ -1202,6 +1219,7 @@ Entry::PlaceholderType Entry::placeholderType(const QString& placeholder) const
         {QStringLiteral("{PASSWORD}"), PlaceholderType::Password},
         {QStringLiteral("{NOTES}"), PlaceholderType::Notes},
         {QStringLiteral("{TOTP}"), PlaceholderType::Totp},
+        {QStringLiteral("{SIZE}"), PlaceholderType::Size},
```

Comment:
There is no need for a reference placeholder. Plus size is not an attribute.

Reply:
Removed placeholder code